frontend/package.json

```diff
@@ -162,7 +162,7 @@
     "test": "jest --coverage --passWithNoTests --coverageReporters json clover",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:ci": "yarn wait-on -t 300000  -lv https://localhost:8000/login && yarn cy:run --config baseUrl=https://localhost:8000"
+    "cy:ci": "yarn wait-on -t 300000  -lv http://localhost:8000/login && yarn cy:run --config baseUrl=http://localhost:8000"
```

Comment:
We should run the e2e test on https. Just to make sure nginx's https is running correctly.